### PR TITLE
ft(comments): create comment model and migration

### PR DESCRIPTION
**Pull request by XROLE**
#### What does this PR do?
- Creates model and migration for comments section
#### Description of Task to be completed?
- [x] Create comment model
- [x] create comment migration
#### How should this be manually tested?
The `comments model` belongs to `article model` so to test this, article model need to be created first. Then,
-  Clone the repo and run `npm run db:migrate`
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
[Finishes #163114833]
#### Screenshots (if appropriate)
- None
#### Questions:
- None

### CODE REVIEW

File: models/comment.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    },
+  });
+  Comment.associate = (models) => {
+    Comment.belongsTo(models.article, {
```
Comment (oreoluwade) [inline models/comment.js]: Change the `article` here to the appropriate case we agreed on.

Also, pull from develop, and uncomment the commented section in the `Article` model

**Comment by oreoluwade** [comment]
Please, change the last 5 digits of your migration file to 23303

File: migrations/20190107123192-create-comment.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('comments', {
```
Comment (oreoluwade) [inline migrations/20190107123192-create-comment.js]: We agreed on the first character of the Model being uppercase. Please change this.

Comment (XROLE) [reply migrations/20190107123192-create-comment.js]: Ok. Right away

File: models/comment.js (hunk 1 above)
Comment (XROLE) [reply models/comment.js]: Feedback implemented

**Review by oreoluwade — APPROVED** [review]

**Pull request merged**